app/controllers/public.js

```diff
@@ -7,6 +8,9 @@ export default Controller.extend({
       return this.get('session.currentRouteName') !== 'public.index';
     }
   }),
+  displayEndDate: computed('model.startsAtDate', 'model.endsAtDate', function() {
+    return !(moment(this.model.startsAtDate).isSame(this.model.endsAtDate, 'minute'));
```

Comment:
Why the additional bracket ?

Reply:
Just to ensure the `!` negate property ,